scipy/spatial/_spherical_voronoi.py

```diff
@@ -216,16 +141,26 @@ class SphericalVoronoi:
 
     """
 
-    def __init__(self, points, radius=None, center=None, threshold=1e-06):
+    def __init__(self, points, radius=1, center=(0, 0, 0), threshold=1e-06):
+        if radius is None:
+            warnings.warn('`radius` is `None`. '
+                          'This will raise an error in a future version. '
+                          'Please provide a floating point number '
+                          '(i.e. `radius=1`).',
+                          DeprecationWarning)
```

Comment:
@rgommers is knowledgeable about these matters. I'm happy if this is an ok way to "deprecate."

Reply:
This is fine I think. Should only be triggered if users explicitly put in `radius=None`, which would be weird. So I don't think this will both anyone.